README.md

```diff
@@ -1,11 +1,30 @@
 # Etcd
 
-> This charm was built from layers! To see the developer docs see `hacking.md` in
-the charm root.
+Etcd is a highly available distributed key value store that provides a reliable way to store data across a cluster of machines. Etcd gracefully handles master elections during network partitions and will tolerate machine failure, including the master.
+
```

Comment:
I personally prefer text wrapped <=79 columns (e.g. to read more easily in gitk).